inflect.py

```diff
@@ -83,10 +88,7 @@ class BadGenderError(Exception):
     pass
 
 
-try:
-    __version__ = __import__("importlib_metadata").version("inflect")
-except Exception:
```

Comment:
By no longer trapping the exception, the library might fail to import if importlib_metadata is missing or if there is an issue resolving the metadata (such as if the package was rolled into an environment that doesn't yet support importlib.metadata's distribution finders, or if the metadata is missing). Probably this library should remain lenient in those cases.